starknet_devnet/devnet_config.py

```diff
@@ -210,6 +218,22 @@ def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, value)
 
 
+class WarnIfDeprecatedArgumentAction(argparse.Action):
+    """
+    Action to warn if user uses old flag;
+    """
+    def __init__(self, nargs=0, **kw):
+        super().__init__(nargs=nargs, **kw)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        if option_string == "--hide-predeployed-accounts":
+            warn(
+                "WARNING: Argument --hide-predeployed-accounts is deprecated\n",
```

Comment:
```suggestion
                "WARNING: Argument --hide-predeployed-accounts is deprecated; applying --hide-predeployed-contracts instead",
```
Also, why the `\n`?

Reply:
To highlight warning. Fixed

Reply:
You don't agree with the other part of the suggestion?